ReactiveCocoa/ObjC+RuntimeSubclassing.swift

```diff
@@ -110,7 +110,7 @@ private func subclassName(of class: AnyClass) -> String {
 ///   - class: The class to swizzle.
 ///   - perceivedClass: The class to be reported by the methods.
 private func replaceGetClass(in class: AnyClass, decoy perceivedClass: AnyClass) {
-	let getClass: @convention(block) (Any) -> AnyClass = { _ in
```

Comment:
<img width="248" alt="screen shot 2017-08-10 at 1 54 52 am" src="https://user-images.githubusercontent.com/11806295/29136168-fa102334-7d6e-11e7-953a-06c63166c47d.png">

Presumably the bridging call relies on `-class` and `+class`, which in turns require bridging due to the use of `Any`.